### PR TITLE
Add __version__ endpoint.

### LINKED ISSUE #35: Add __version__

### PR DESCRIPTION
**Pull request by Natim**
Fixes #35

### CODE REVIEW

File: pollbot/views/utilities.py [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
 
 from aiohttp import web
 import ruamel.yaml as yaml
 
 HERE = os.path.dirname(__file__)
 
 
+async def version(request):
+    # Use the version.json file in the current dir.
+    with suppress(IOError):
```
Comment (magopian) [inline pollbot/views/utilities.py]: When would we have no `version.json` file? I don't understand what this endpoint is doing, is it returning the version of the API?

**Comment by Natim** [comment]
Good point, it is the same as what we did for https://github.com/Kinto/kinto/pull/747
The plan is to have the commit number of the running version as well as the tag. The version.json file is created during the Docker image creation and we serve it through an endpoint.

I realized looking for information that @leplatrem did a second PR to make the version file path configurable. I will probably add a env variable to make it configurable as well.

File: pollbot/views/utilities.py (hunk 1 above)
Comment (Natim) [inline pollbot/views/utilities.py]: Using the Makefile when running tests, tests-once or serve command.
In production [during the Docker image creation](https://github.com/mozilla/PollBot/pull/36/files#diff-29944324a3cbf9f4bd0162dfe3975d88R28)

**Comment by magopian** [comment]
Ok I understand now, but does it mean that not having a version.json file should make the service fail? Instead of suppressing the exception?

**Comment by Natim** [comment]
If the file is missing we don't expose the endpoint instead it return a 404.

**Pull request merged**